platform/marvell-armhf/sonic-platform-nokia/7215/sonic_platform/thermal.py

```diff
@@ -226,6 +233,16 @@ def get_high_critical_threshold(self):
 
         return float("{:.3f}".format(thermal_high_crit_threshold))
 
+    def get_minimum_recorded(self):
+        if self._minimum is None:
+           self.get_temperature()
```

Comment:
Do we have any other chance to update the min/max temperature?
Maybe we can always invoke get_temperature and then return the min/max temperature

Reply:
I'm not sure what you are getting at here.
The min and max temp values are set/adjusted within each call to get_temperature()
The logic here is simply to account for the case where get_minimum_recorded() is called before the first call to get_temperature(). By calling get_temperature() here we ensure that the self._minimum variable is set to a temperature value.

Reply:
If no one call `get_temperature` then the min/max value will never have a chance to update.

I checked other implementation and they will call get_temperature and try refresh the min/max during get min/max value. e.g. https://github.com/Azure/sonic-buildimage/blob/e5c2937faec615367a04075ab1a4dc7aeeee1be2/platform/broadcom/sonic-platform-modules-ruijie/b6510-48vs8cq/sonic_platform/thermal.py

Reply:
I see your point. By my thinking get_temperature() will be called regularly by the traditional thermalctld daemon used on our platform but I will make the change as you suggest. FWIW, I modeled my approach after an earlier version of Arista platform code. I can see that in their latest version they now use the approach that you suggest.